.github/workflows/kind.yml

```diff
@@ -15,20 +15,13 @@ defaults:
     shell: bash
 
 jobs:
-  kind-tests:
+  quick-checks:
+    name: Quick Checks
     runs-on: ubuntu-latest
     env:
+      KIND_VERSION: latest
```

Comment:
I don't _think_ this is necessary since it's the default--is it just here for transparency?

Reply:
Yep, for transparency, especially if the "no strategy.matrix" comment is removed.